avocado/plugins/spawners/podman.py

```diff
@@ -1,11 +1,9 @@
 import asyncio
 import json
 import os
-import re
```

Comment:
There's a chicken and egg problem in the proposal given in the commit message, specifically on this part:

```
With the introduction of Package Requirements, the Avocado Software can be
handled as a requirement of `type="package"` if it is available in the
distro repository and if it is needed to run the test.
```

That's not a viable alternative, because for `avocado-runner-requirement-software` to run (supposedly to install Avocado), Avocado itself needs to be installed.  I know it's not time to decide what the most basic requirements are, but we should at least not add that as a proposal.

So, just as an example, maybe we'll settle on Python 3 + setuptools + pip as the basic requirements for a "avocado-runner-requirement-core-avocado" runner.

I think an action item here is to reword the commit message to reflect that.

Reply:
I don't see anything more productive than thinking about software architecture in the shower, even during weekends :D